Tribler/Core/Modules/restapi/events_endpoint.py

```diff
@@ -88,9 +121,9 @@ def render_GET(self, request):
         """
         self.events_request = request
 
-        request.write(json.dumps({"type": "events_start"}))
+        request.write(json.dumps({"type": "events_start"}) + '\n')
```

Comment:
It most definitely is a method/trick for parsing. I see the problem though. if this works on all platforms I am fine with it, but I guess it should be documented somewhere?